sdk/translation/azure-ai-documenttranslator/README.md

```diff
@@ -1,5 +1,11 @@
 # Azure Document Translator client library for Java
 
+## Deprecation Notice
```

Comment:
We also need to update the description in the pom.xml and changelog and release the deprecated package. See [deprecation guidelines for Java](https://github.com/Azure/azure-sdk-for-java/blob/main/doc/deprecation_process.md)